src/components/NavBar.jsx

```diff
@@ -0,0 +1,27 @@
+import { NavLink, Link } from 'react-router-dom';
+import logo from '../images/planet_1.png';
+
+const NavBar = () => (
+  <header>
+    <nav>
+      <Link to="/">
+        <img className="logo" src={logo} alt="" />
+        <h1>Space traveler&apos; Hub</h1>
+      </Link>
+      <ul>
+        <NavLink className="nav-items" to="/">
+          Rockets
+        </NavLink>
+        <NavLink className="nav-items" to="/missions">
+          Missions
+        </NavLink>
+        <span>|</span>
+        <NavLink className="nav-items" to="/profile">
+          My Profile
+        </NavLink>
+      </ul>
+    </nav>
+  </header>
+);
```

Comment:
[Optional] It is better to use a <Outlet> context API in parent route elements to render their child route elements. This allows nested UI to show up when child routes are rendered. [React Outlet](https://reactrouter.com/en/main/components/outlet)

Reply:
@cherelemma thank you.